Documentation/CodeSnippets/ExtensionConfiguration/SitePackage-settings.rst.txt

```diff
@@ -7,9 +7,9 @@
 
     styles:
       templates:
-        layoutRootPath: EXT:site_package/Resources/Private/Layouts/ContentElements
-        partialRootPath: EXT:site_package/Resources/Private/Partials/ContentElements
-        templateRootPath: EXT:site_package/Resources/Private/Templates/ContentElements
+        layoutRootPath: EXT:site_package/Resources/Private/ContentElements/Layouts
```

Comment:
As already discussed, there is no clear guideline at the moment which folder structure is the "best one", at least not to my knowledge. However, this looks reasonable to me.

Reply:
Ack